architecture/core.rst

```diff
@@ -428,7 +428,31 @@ MMU
 ***
 Translation tables
 ==================
-OP-TEE uses several L1 translation tables, one large spanning 4 GiB and two or
+
+OP-TEE supports two translation table formats:
+
+1. short-descriptor translation table format, availble on ARMv7-A and
+   ARMv8-A AArch32
+2. long-descriptor translation format, available on ARMv7-A with LPAE and
+   ARMv8-A
```

Comment:
"Short descriptor format, availble...", "Long descriptor format, available..."
"LPAE (Large Physical Address Extension)" then drop the definition below.
s/availble/available/